app/page/stop-map.cjsx

```diff
@@ -20,15 +20,16 @@ class Page extends React.Component
       @context.executeAction FavouriteStopsAction.addFavouriteStop, @props.params.stopId
 
     <DefaultNavigation className="fullscreen">
-      <Map lat={@props.stop.lat+0.0005}
+      <StopCardHeader stop={@props.stop} favourite={favourite} addFavouriteStop={addFavouriteStop} headingStyle="h3" className="stop-page" infoIcon={true}/>
+      <Map lat={@props.stop.lat}
            lon={@props.stop.lon}
-           zoom={18}
+           zoom={16}
```

Comment:
Checked with Petro: it's ok to make zoom smaller (so that you see slightly more context).